.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+- [x] Add optional property support
+- [x] Fix memory leaks
+- [ ] Add GitHub issues for OrderingType investigation, physical prune stage, and TPL break statement
+
+## Performance
+If your PR has the potential to greatly affect the performance of the system, please address these by benchmarking your changes with respect to master, or profiling the performance. You may do this in one of the following ways:
+1. Inline a table outlining performance results. An [example](https://github.com/cmu-db/noisepage/pull/1109) from a PR @gonzalezjo follows:
+
+    | Machine Type 	| Terminals 	| Scale Factor 	| Socket Type 	| Transactions / Second 	|
+    |--------------	|-----------	|--------------	|-------------	|-----------------------	|
+    | Bare metal   	| 10        	| 10           	| UNIX        	| 8346 (+34%)           	|
+    | Bare metal   	| 10        	| 10           	| INET        	| 6214                  	|
+    | Bare metal   	| 1         	| 1            	| UNIX        	| 914 (+37%)            	|
+    | Bare metal   	| 1         	| 1            	| INET        	| 668                   	|
+    | VMware       	| 10        	| 10           	| UNIX        	| 3728 (+24%)           	|
+    | VMware       	| 10        	| 10           	| INET        	| 3005                  	|
+    | VMware       	| 1         	| 1            	| UNIX        	| 289 (+28%)            	|
+    | VMware       	| 1         	| 1            	| INET        	| 226                   	|
+
+2. Create a Google Sheets document noting baseline performance and scalability, as is done [here](https://docs.google.com/spreadsheets/d/1eng7O98KaG0fJn6SVavquPr2zipzp5Ju9FtQylHPV2o/edit?usp=sharing) in an example from @mbutrovich
+3. Create an SVG of profiling results per the [EC2 profiling instructions](https://github.com/cmu-db/noisepage/wiki/Profiling-on-EC2), as is done [here](https://drive.google.com/file/d/1xSn1o7RyazbvnyKjlxMets1H7gQlQ2WO/view?usp=sharing) in an example from @thepinetree
+
+## Further Work
+If your PR unlocked the potential for further improvement, please note them here and create additional issues! Do the same if you discovered bugs in the process of development. An [example](https://github.com/cmu-db/noisepage/pull/1109) from a PR by @gonzalezjo follows:
+
+### Investigating loopback and TCP overhead
+
+This is probably a dead end, but less of a dead end than libevent/epoll stuff.
+
+I'd like to work on improving our INET socket overhead, but at this point, that might not be doable without being a kernel engineer. Nothing I tried measurably reduced loopback and/or INET overhead, and I tried a lot. Still, I think it's worth digging some more.
+
+One question I have is how much of the speedup comes from avoiding TCP, and how much comes from using a glorified pipe instead of loopback. This would be interesting to know, but I have no idea how I'd measure, and I'm not convinced that I'd be able to make anything useful from the answer.
```

Comment:
Suggestion: add a template at the end.

## Template

```
# Heading

## Description

## Remaining tasks

- [ ] Foo
- [ ] Bar
- [ ] Baz

## Performance

## Further work

```

Reply:
Added